experiments/AMIP/components/ocean/slab_ocean_init.jl

```diff
@@ -36,30 +36,33 @@ struct OceanSlabParameters{FT <: AbstractFloat}
 end
 name(::SlabOceanSimulation) = "SlabOceanSimulation"
 
-# init simulation
-function slab_ocean_space_init(::Type{FT}, space, p) where {FT}
+function get_T_sfc(params, coord)
```

Comment:
moved code into separate function to use `broadcast` instead of `map` on `coords`